examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -46,13 +45,7 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
             }
         }
 
-    private val successfulPurchaseCallback: (purchase: StoreTransaction, customerInfo: CustomerInfo) -> Unit =
-        { storeTransaction, _ ->
-            toggleLoadingIndicator(false)
-            handleSuccessfulPurchase(storeTransaction.orderId)
-        }
-
-    private val successfulUpgradeCallback: (purchase: StoreTransaction?, customerInfo: CustomerInfo) -> Unit =
+    private val successfulPurchaseCallback: (purchase: StoreTransaction?, customerInfo: CustomerInfo) -> Unit =
```

Comment:
during development/until i figure out the deprecation/renaming strategy, i'm just changing this type.